downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/pixels/DownloadsPixelName.kt

```diff
@@ -23,4 +23,6 @@ enum class DownloadsPixelName(override val pixelName: String) : Pixel.PixelName
     DOWNLOAD_REQUEST_SUCCEEDED("m_download_request_succeeded"),
     DOWNLOAD_REQUEST_FAILED("m_download_request_failed"),
     DOWNLOAD_REQUEST_CANCELLED("m_download_request_cancelled"),
+    DOWNLOAD_REQUEST_CANCELLED_BY_USER("m_download_request_cancelled_by_user"),
+    DOWNLOAD_REQUEST_RETRIED("m_download_request_retried"),
```

Comment:
We're currently accessing the `download-impl` here, which should not happened. Will fix this later on